priv/templates/phx.gen.json/fallback_controller.ex

```diff
@@ -10,15 +10,15 @@ defmodule <%= inspect context.web_module %>.FallbackController do
   def call(conn, {:error, %Ecto.Changeset{} = changeset}) do
     conn
     |> put_status(:unprocessable_entity)
-    |> put_view(<%= inspect context.web_module %>.ChangesetView)
+    |> put_view(html: <%= inspect context.web_module %>.ChangesetView, json: <%= inspect context.web_module %>.ChangesetView)
```

Comment:
I think those changes are not necessary given our new approach that `:_` is always first?